pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -135,8 +140,42 @@ func (b *ConsoleServerCLIConfigBuilder) StatusPageID(id string) *ConsoleServerCL
 	return b
 }
 
-func (b *ConsoleServerCLIConfigBuilder) OAuthServingCert() *ConsoleServerCLIConfigBuilder {
-	b.CAFile = oauthServingCertFilePath
+func (b *ConsoleServerCLIConfigBuilder) AuthConfig(authnConfig *configv1.Authentication, caConfigMap *corev1.ConfigMap) *ConsoleServerCLIConfigBuilder {
+	switch authnConfig.Spec.Type {
+	case "", configv1.AuthenticationTypeIntegratedOAuth:
+		b.oauthClientID = api.OAuthClientName
```

Comment:
here we are assuming that the `b.authType` is defaulting to `openshift` ?
right @stlaz

Reply:
it does, but we can/should be explicit